eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -737,18 +737,6 @@
           "code" : "java.method.removed",
           "old" : "method com.azure.resourcemanager.keyvault.models.VaultCheckNameAvailabilityParameters com.azure.resourcemanager.keyvault.models.VaultCheckNameAvailabilityParameters::withType(java.lang.String)",
           "justification": "The property is a constant, which should not be changed."
-        },
-        {
-          "ignore": true,
-          "code" : "java.method.removed",
-          "old" : "method com.azure.resourcemanager.containerservice.models.ManagedClusterLoadBalancerProfile com.azure.resourcemanager.containerservice.models.ManagedClusterLoadBalancerProfile::withEffectiveOutboundIPs(java.util.List<com.azure.resourcemanager.containerservice.models.ResourceReference>)",
-          "justification": "The effectiveOutboundIPs was never a client-side modifiable property."
-        },
-        {
-          "ignore": true,
-          "code" : "java.method.removed",
-          "old" : "method com.azure.resourcemanager.containerservice.models.ManagedClusterNatGatewayProfile com.azure.resourcemanager.containerservice.models.ManagedClusterNatGatewayProfile::withEffectiveOutboundIPs(java.util.List<com.azure.resourcemanager.containerservice.models.ResourceReference>)",
-          "justification": "The effectiveOutboundIPs was never a client-side modifiable property."
```

Comment:
Be careful, wrong deletion.

Reply:
fixed by [9fb4296](https://github.com/Azure/azure-sdk-for-java/pull/44748/commits/9fb4296da7d5efaaaf05b524eb93067b1cdf69c9)